src/components/ScrollWheel/Knob.tsx

```diff
@@ -39,33 +45,19 @@ const CenterButton = styled.div<{ size: number }>`
   width: ${(props) => props.size / 2.5}px;
   height: ${(props) => props.size / 2.5}px;
   border-radius: 50%;
-  box-shadow: rgb(191, 191, 191) 0px 1em 3em inset;
-  background: rgb(225, 225, 225);
-  border: 1px solid #b9b9b9;
+  box-shadow: ${({ deviceColor }) =>
+      deviceColor === 'silver' ? 'rgb(191, 191, 191)' : 'rgb(50, 50, 50)'}
+    0px 1em 3em inset;
+  background: ${({ deviceColor }) =>
+    deviceColor === 'silver' ? '#ffffff' : '#7d7c7d'};
```

Comment:
I'm thinking we'll want a more generalized way to add device colors here in the future, but this is definitely a good place to start

Reply:
Added themes in a recent commit :)